### DISCUSSION TITLE
BaselHack chat

### DISCUSSION THREAD

**Discussion by culmat** [BaselHack/start-me-up#1] [Announcements]
Hii there

**Comment by vic-codecamp** [comment]
yo

**Reply by AgaSieradzka** [reply]
great! (this is Aga)

**Comment by culmat** [comment]
What are existing / similar platform intiatives?
Can we have partnerships / reuse data?
 Can we make start-me-up available to others ( other contaons / cities )?

**Comment by djurdja2021** [comment]
[Start-me-up Präsentation BaselHack.pptx](https://github.com/BaselHack/start-me-up/files/7147969/Start-me-up.Prasentation.BaselHack.pptx)

**Reply by djurdja2021** [reply]
![image](https://user-images.githubusercontent.com/90504886/132983797-38f48951-914f-4e84-8955-d60c8c7e15d2.png)

**Comment by culmat** [comment]
presentation link 
https://docs.google.com/presentation/d/1VXHxtw7YUvxUoZdSCGlIadoWvd5UXOiEwRttoII3a3Q/edit#slide=id.gef3d15f68e_0_206

**Comment by AgaSieradzka** [comment]
Time To Live (TTL)  concept that defines the expiration timeline, guards the accuracy and prevents outdated content to remain published. 
It works on the basis of tsix-month timeframe, ie. requires the Registered Users to validate accuracy of their entry every 6 months - in order not to be deleted/ceased visibility from the StartMeUp page. 
Community members are comprised of the groups, with their access privileges defined as follows:
1.	Moderators (edits are effective immediately)
a.	Delete Users 
b.	Edit All entries
c.	Change TTL priviledges

2.	Registered Users (edits are effective immediately)
a.	Create entry 
b.	Edit own entry 
c.	Claim       (??make available on Google search)
d.	Report     (?? On which data???)


3.	Visitors (any)
a.	Filter data
b.	Receive newsletter
c.	?? be prompted on changes, information about new Registered users
